starboard/doc/evergreen/evergreen_binary_compression.md

```diff
@@ -41,56 +44,57 @@ This feature is incompatible with the Memory Mapped file feature that is
 controlled by the `--loader_use_mmap_file` switch. With compression, we lose the
 required 1:1 mapping between the file and virtual memory. So,
 `--loader_use_mmap_file` should not be set if compression is enabled for any
-installation slots (next section).
+installation slots.
 
-## Enabling Binary Compression
+## Disabling Binary Compression
 
-Separate steps are required in order for a partner to enable compression for a)
+Before proceeding, please note that binary compression should not be disabled in
+production.
```

Comment:
Add:

Compression is required starting from Cobalt 25. The feature of disabling binary compression should only be used for debugging or emergency.

Reply:
Done